docs/actions/introduction.md

```diff
@@ -0,0 +1,171 @@
+# Actions
```

Comment:
you use Action instead of Task. If we choose to use action, we should change the interface name in order to be consistent

Reply:
Indeed, this is a deliberated peacefull naming revolution :)